package haptix.comm.msgs;

message hxEmpty
{
  optional bool unused = 1;
}
